### ISSUE TITLE
Message not fully readable (classicchat)

### ISSUE THREAD

**Issue by svenom** [Miei/TOS-lua#20]
Sometimes the Messages arnt fully readable and the first line cuts off.

Happends 100% inside Dungeons since the party name got filled with spaces (since the last ToS Update.)
Had to disable the Addon since i could read any message wrote in party chat.

+ Sometime the Messages got cut in other cases.

![party](https://cloud.githubusercontent.com/assets/19372864/15793093/f52dadb4-29e1-11e6-8a2a-259d96702e77.png)

**Comment by Miei** [comment]
Noticed that, but it happens with names on their character info too, so I don't know exactly what I'm supposed to do about it.

If you can give me some scenarios that the message cuts off in then I can take a look.

**Comment by Excrulon** [comment]
I think it might be because you need to resize the component. If the component is smaller than the string, it gets cut off. Or not, dunno.

**Comment by svenom** [comment]
I renabled the Plugin and gonna collect some Screens. 
It often happens with linked Items afaik (but this happens with the vanilla chat too).

But this Dungeon scenarios isnt playable :(

**Comment by Miei** [comment]
Yeah, there are some errors in their textbox resizing with item links. Afaik resizing doesn't help with that, since then the rest of the text won't fill the chatbox.

And yeah, I'll update to remove the space in names, not sure why the game is passing them like that in the first place though.

**Comment by svenom** [comment]
So -  i got you 2 more Screens. As you can see, the Dungeon Chat isnt readable.

I dont know how far you can access the Vars, but cant you just trim the Playername since names are only alphanumeric?

![example](https://cloud.githubusercontent.com/assets/19372864/15799944/bd03dd78-2a6d-11e6-991e-5f76998690c8.png)
![example2](https://cloud.githubusercontent.com/assets/19372864/15799945/bd04c422-2a6d-11e6-96bb-5c4ad09078fd.png)
